tt_metal/impl/dispatch/command_queue.hpp

```diff
@@ -186,17 +191,37 @@ namespace detail{
 
 class CommandQueue {
    public:
-    CommandQueue(Device* device);
+    CommandQueue(Device* device, uint32_t command_queue_channel);
```

Comment:
Can command_queue_channel be inferred somehow? This is something user will be creating more often going forward

Reply:
hope this will be addressed in the follow-on PR?

Reply:
Good point. I didn't think about this. Yes, I can address in a follow-up.